notebooks/202-facies-segmentation/README.md

```diff
@@ -0,0 +1,90 @@
+# Facies segmentation Python Demo
+
+This demo demonstrate how to run facies classification using OpenVINO&trade;
+
+This model came from seismic interpretation tasks. Fasies is the overall characteristics of a rock unit that reflect its origin and differentiate the unit from others around it.  Mineralogy and sedimentary source, fossil content, sedimentary structures and texture distinguish one facies from another. Data are presented in the 3D arrays.
+
+
+## How It Works
+Upon the start-up, the demo application loads a network and an given dataset file to the Inference Engine plugin. When inference is done, the application displays 3d itkwidget viewer with facies interpretation.
+
+## Running
+
+### Installation of dependencies
+First of all, you need to create the required CPU extension, just follow the instructions below:
+
+Steps to create CPU extension:
+
+```bash
+$ source <openvino_install>/bin/setupvars.sh
+$ export TBB_DIR=${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/tbb/cmake/
+
+$ cd ./user_ie_extensions
+$ mkdir build && cd build
+$ cmake .. -DCMAKE_BUILD_TYPE=Release && make -j$(nproc --all)
+$ mv libunpool_cpu_extension.so ../../
+```
```

Comment:
Do you mind to move all these steps to the notebook? So user will have everything in a single place